spec/controllers/v1/actions_controller_spec.rb

```diff
@@ -14,7 +15,11 @@
         post :create, person: { email: 'joe@example.com' }, template_id: 'real_id'
         parsed_response = JSON.parse(response.body)
         expect(parsed_response.slice('uuid', 'completed_activities').values)
-          .to eq ["the-uuid", ["real_id"]]
+          .to eq ['the-uuid', ['real_id']]
+        activities = parsed_response['activities']
+        expect(activities.count).to eq 2
+        expect(activities.first.keys).to eq ['name', 'order', 'completed', 'template_id']
+        expect(activities.map{|a| a['completed']}).to eq [true, false]
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.